File: jsfring-webservice/src/main/java/fr/pinguet62/jsfring/webservice/dto/ProfileDto.java
package fr.pinguet62.jsfring.webservice.dto;

import fr.pinguet62.jsfring.model.sql.Profile;
import lombok.Data;

import java.util.Set;

/**
 * @see Profile
 */
@Data
public class ProfileDto {

    private int id;

    private Set<String> rights;

    private String title;

}

File: jsfring-webservice/src/main/kotlin/fr/pinguet62/jsfring/webservice/dto/ProfileDto.kt
package fr.pinguet62.jsfring.webservice.dto

import fr.pinguet62.jsfring.model.sql.Profile

/**
 * @see Profile
 */
data class ProfileDto(
        var id: Int?,
        var rights: Set<String>?,
        var title: String?
) {
    constructor() : this(null, null, null)
}
